lib/engine/game/g_1846.rb

```diff
@@ -330,7 +336,24 @@ def event_remove_tokens!
       end
 
       def bankruptcy_limit_reached?
-        @bankruptcies >= @players.size - 1
+        @bankrupt_players.size >= (all_players.size - 1)
+      end
+
+      def sellable_bundles(player, corporation)
+        return [] if corporation.owner == @share_pool
+
+        super(player, corporation)
```

Comment:
you can just call super it passes teh args

Reply:
I remember running into trouble with that once, probably a case where I wanted different args passed in. 

71fcf964